src/Test.FunctionalTests.Sarif/MultitoolCommandLineTests.cs

```diff
@@ -35,5 +36,64 @@ public void Multitool_LaunchesAndRunsSuccessfully()
                 process.ExitCode.Should().Be(0);
             }
         }
+
+#if DEBUG
+        [Fact]
+        [Trait(TestTraits.WindowsOnly, "true")]
+        public void Multitool_LaunchesAndRunsSuccessfully_With12kFiles()
+        {
+            using var assertionScope = new AssertionScope();
+
+            string multitoolPath = Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(),
+                @"..\..\Sarif.Multitool\netcoreapp3.1\Sarif.Multitool.exe"));
+
+            string directoryPath = Path.Combine(Path.GetTempPath(), "SarifMultitool12kTestFiles");
+             
+            if (Directory.Exists(directoryPath))
+            {
+                Directory.Delete(directoryPath, true);
+            }
+
+            Directory.CreateDirectory(directoryPath);
+
+            for (int i = 1; i <= 12000; i++)
+            {
+                string filename = $"file_{i}.txt";
+                string filepath = Path.Combine(directoryPath, filename);
+                File.WriteAllText(filepath, " ");
+            }
+
+            var startInfo = new ProcessStartInfo(multitoolPath, $@"analyze-test {directoryPath}\*")
+            {
+                WindowStyle = ProcessWindowStyle.Hidden,
+                CreateNoWindow = true,
+                RedirectStandardOutput = true,
+                UseShellExecute = false
+            };
+
+            using (var process = Process.Start(startInfo))
+            {
+                var timer = new System.Timers.Timer(30000); // 30 seconds.
```

Comment:
with fix, it will pass, it will only runs within 5 seconds so will not affect testing run total time:
![image](https://github.com/microsoft/sarif-sdk/assets/81775155/9095d703-33ee-4104-b6dc-d82ebac80314)

before fix, it will fail and the message will be like below for dev:
![image](https://github.com/microsoft/sarif-sdk/assets/81775155/a2d57290-1b93-49bd-9afb-db58951b3fe4)